pystac/__init__.py

```diff
@@ -3,59 +3,72 @@
 """
 
 # flake8: noqa
-
-
-class STACError(Exception):
-    """A STACError is raised for errors relating to STAC, e.g. for
-    invalid formats or trying to operate on a STAC that does not have
-    the required information available.
-    """
-    pass
-
-
-from pystac.version import (__version__, get_stac_version, set_stac_version)
-from pystac.stac_io import STAC_IO
-from pystac.extensions import Extensions
-from pystac.stac_object import (STACObject, STACObjectType)
-from pystac.media_type import MediaType
-from pystac.link import (Link, HIERARCHICAL_LINKS)
-from pystac.catalog import (Catalog, CatalogType)
-from pystac.collection import (Collection, Extent, SpatialExtent, TemporalExtent, Provider)
-from pystac.item import (Item, Asset, CommonMetadata)
-
-from pystac.serialization import stac_object_from_dict
+from pystac.errors import (
+    STACError,  # type:ignore
+    STACTypeError,  # type:ignore
+    RequiredPropertyMissing,  # type:ignore
+)
+
+from typing import Any, Dict, Optional
+from pystac.version import (
```

Comment:
Please add a comment explaining why the order of imports isn't alphabetical within the pystac imports